package.json

```diff
@@ -341,6 +341,7 @@
     "react-dom": "^18.2.0",
     "react-i18next": "^13.0.0",
     "react-router-dom": "^6.3.0",
+    "read-last-lines": "^1.8.0",
```

Comment:
As alternative to adding new dependency `read-last-lines` I have found that Winston logger that we use has a `query()` api for querying logs: https://github.com/winstonjs/winston#querying-logs

If we extend our `LoggerService` api with query function we might be able to get away with it.